src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -348,6 +384,37 @@ private void EnsureSignature()
             }
         }
 
+        // PROTOTYPE(semi-auto-props): Consider renaming to EnsureBackingFieldIsSynthesized.
+        // PROTOTYPE(semi-auto-props): Consider not using CompletionPart for this purpose.
+        private void EnsureAccessorsBinding()
+        {
+            if (!_state.HasComplete(CompletionPart.FinishPropertyAccessorsBinding))
+            {
+                if (_state.NotePartComplete(CompletionPart.StartPropertyAccessorsBinding))
+                {
+                    // Ensure the binding is done so we guarantee that we have the BackingField set if necessary.
+                    if (_lazyBackingFieldSymbol is null && this is SourcePropertySymbol propertySymbol)
+                    {
+                        if (propertySymbol.GetMethod is SourcePropertyAccessorSymbol { ContainsFieldKeyword: true } getMethod)
+                        {
+                            var binder = getMethod.TryGetBodyBinder();
+                            binder?.BindMethodBody(getMethod.SyntaxNode, BindingDiagnosticBag.Discarded);
+                        }
+                        if (propertySymbol.SetMethod is SourcePropertyAccessorSymbol { ContainsFieldKeyword: true } setMethod)
```

Comment:
>propertySymbol.SetMethod is SourcePropertyAccessorSymbol { ContainsFieldKeyword: true } setMethod [](http://example.com/codeflow?start=28&length=97)

Is it possible that we already created the backing field while binding the getter?

Reply:
Fixed.